Add `UKCCX` tournament article (#7272)
ppy/osu-wiki: wiki/Tournaments/UKCC/X/en.md

# UK Community Cup X

> ![UKCCX logo](img/logo.jpg)

Reviewer: banner

See the [Pocket Picks sheet](https://docs.google.com/spreadsheets/d/1Jj9t9Fn_WoJmIYW6dGfDgL4djEZfAIMQcDgfV-mskmc/edit#gid=1724187283) for a breakdown of the participants Pocket Picks.

## Match results

### Final Round

#### Day 17

| Player 1 |  |  | Player 2 | Match link | Player status |
| --: | :-: | :-: | :-- | :-- | :-- |
| **Kingling** ![][flag_GB] | **4** | 1 | ![][flag_GB] Malteser | [#1](https://osu.ppy.sh/community/matches/80493729) | *Malteser was eliminated* |
| **Voss** ![][flag_GB] | **7** | 1 | ![][flag_GB] Kerantina | [#1](https://osu.ppy.sh/community/matches/80499466) |  |

#### Grand Finals: Part 1

The full results for this match can be found at this [link](https://osu.ppy.sh/community/matches/80587587).

| Players | Score | Player status |
| --: | :-: | :-- |
| ![][flag_GB] Kingling | 4 |  |
| ![][flag_GB] Voss | 0 |  |
| ![][flag_GB] Kerantina | 0 | *Kerantina was eliminated* |

#### Grand Finals: Part 2

| Player 1 |  |  | Player 2 | Match link | Player status |
| --: | :-: | :-: | :-- | :-- | :-- |
| **Voss** ![][flag_GB] | **9** | 6 | ![][flag_GB] Kingling | [#1](https://osu.ppy.sh/community/matches/80679009) | *Voss was eliminated* |

### Round 8

> #### Day 15

Reviewer 2: i think it may be better to replace the `Day` headers with something like `Sunday, 2000-12-12:`, as currently it seems pretty meaningless as it is

Reviewer: yeah there's no need to deviate from the template here

![image](https://user-images.githubusercontent.com/36758269/166327562-3091a3dd-7116-43c7-afff-62d0dd756e8d.png)

Author: Yeah, that could work. I went with the `Day` headings to stay in line with how the original spreadsheet was presented but I think the `Day, date:` format works nicer IMO. I'll apply the necessary changes.

> | Player 1 |  |  | Player 2 | Match link | Player status |

Reviewer: "Outcome" instead of "Player status"

while the tables might look a little empty on the github preview, i think it should look ok on the website since the cells aren't boxed all the way around

Top 10 qualification results:

| Placing | Team | Normalised score | Average score |
| --: | :-- | :-- | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761) | 7,398,855 | 822,095 |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_GB] [Voss](https://osu.ppy.sh/users/7657761) | 7,323,703 | 813,745 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_GB] [Jemzsee](https://osu.ppy.sh/users/11553627) | 7,149,778 | 794,420 |
| *4th place* | ![][flag_GB] [Chemistry](https://osu.ppy.sh/users/6069540) | 7,111,662 | 790,185 |
| *5th place* | ![][flag_GB] [Apo11o](https://osu.ppy.sh/users/9558549) | 6,900,462 | 766,718 |
| *6th place* | ![][flag_GB] [Malteser](https://osu.ppy.sh/users/5218178) | 6,887,332 | 765,259 |
| *7th place* | ![][flag_GB] [Defex](https://osu.ppy.sh/users/9674143) | 6,414,683 | 712,743 |
| *8th place* | ![][flag_GB] [Mahmood](https://osu.ppy.sh/users/7627844) | 6,234,206 | 692,690 |
| *9th place* | ![][flag_PH] [LilyFlower](https://osu.ppy.sh/users/7449949) | 6,217,907 | 690,879 |

> | *10th place* | ![][flag_gb] [Doomsday](https://osu.ppy.sh/users/18983) | 5,931,218 | 659,024 |

Author: ```suggestion
| *10th place* | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 5,931,218 | 659,024 |
```

> ## Ruleset

Reviewer: aaa the organisation here is so good

- If it is impossible to find a time to reschedule to that suits both teams, both teams should talk to the hosts (![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) or ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973)) who will attempt to sort something out.

### Pre-match

- Referees will create the lobby approximately **10 minutes** before the match is due to start.
- Players have a 10-minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will receive losses.
- We take no-shows seriously. If you fail to show up to your match without valid reason, you may forfeit your right to participate in future UKCC tournaments.
- The referee cannot play in the match.

### Qualifiers

- Players will play each map in the Qualifiers mappool once.
- There will be 3 NoMod, 2 Hidden, 2 Hard Rock and 2 Double Time maps.
- The top 32 scores will progress into the Swiss bracket.

### Swiss

- Since Swiss bracketing is a natural seeding system, matchups will be drawn between **top 16 and bottom 16 players randomly** to begin.

> - Two matches will happen every weekend for each player, one on Saturday and one on Sunday - therefore, **two matchups will be drawn per weekend** for each player.

Reviewer: ```suggestion
- Two matches will happen every weekend for each player, one on Saturday and one on Sunday—therefore, **two matchups will be drawn per weekend** for each player.
```

- The order the matches are played in **does not matter**.
- Matchups will be drawn between players with the **same aggregate match W-L-D score** (eg. players at 3-0-1 will prioritise playing against the players with the most similar W-L-D), and will prioritise **avoiding rematches** until each active player has played each other once.

### Rolling

- When both players are in the lobby and ready, the referee will ask the players to `!roll`. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).

### Banning

- Each player will have **one** ban.
- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
- There are no restrictions on bans and picks. Players are free to ban/pick any map regardless of mod.

### Warmups

- There will be **no** warmups played.

### Picking

- Players have **90 seconds** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.

> - **Failed scores will count, and every map will be played with the NF modification enabled**

Reviewer: it would be "modifier" and not "modification" anyway

```suggestion
- **Failed scores will count, and every map will be played with the No Fail mod enabled**
```

- Each player picks **four times**. Once 8 maps have been picked and played, the match ends.

> - There are **no Tiebreakers.**

Reviewer: ```suggestion
- There are **no tiebreakers.**
```

### Health system

- Each player who qualifies into Swiss will **begin with 8,000,000 HP**, which carries through matches.
- During a match, the **losing player of each map pick** will **lose HP equal to the score difference** between the two players.

> - If a player’s HP reaches 0, they are **immediately eliminated from the tournament**, and if their match is still in progress it ends immediately.

Reviewer: ```suggestion
- If a player's HP reaches 0, they are **immediately eliminated from the tournament**, and if their match is still in progress it ends immediately.
```

### Pocket Picks

> - When signing up to the tournament, each player will **submit three maps** to be listed as “Pocket Picks” for use during Swiss.

Reviewer: ```suggestion
- When signing up to the tournament, each player will **submit three maps** to be listed as "Pocket Picks" for use during Swiss.
```

- Pocket Picks can be **any map, with any mods**. The mods chosen are forced.

> - While there is room to push boundaries, the **rule of common sense** applies when submitting Pocket Picks - the maps will be reviewed case by case, and if a map is too far out of the playable range it will be rejected and a new Pocket Pick will have to be submitted.

Reviewer: ```suggestion
- While there is room to push boundaries, the **rule of common sense** applies when submitting Pocket Picks—the maps will be reviewed case by case, and if a map is too far out of the playable range it will be rejected and a new Pocket Pick will have to be submitted.
```

> - Only up to one Pocket Pick can be played per match, and only as a player’s **3rd or 4th pick during the match**.

Reviewer: ```suggestion
- Only up to one Pocket Pick can be played per match, and only as a player's **3rd or 4th pick during the match**.
```

- Once a Pocket Pick has been played, it **cannot be picked again for the rest of the tournament**.
- Pocket Picks cannot be banned.

> ### Byes

Reviewer: even after reading these points, i still have absolutely no idea what a "bye" is

Author: Would a rewording of the bullet points to the following be better at helping understand what a "Bye" is?

```
- A player receives a bye when an opponent is eliminated from the tournament before the scheduled match can take place.
- If a player forfeits a match, both players will receive a bye for that match.
- If a player receives a bye, once all scheduled matches for each round are completed, the player will receive an HP deduction of 90% of the average HP lost by the players at their general aggregate match score (+/- 1).
```

Reviewer: ah, so this functions as a sort of passive hp drain to account for matches that weren't played

can the last point be reworded as

```
- If a player receives a bye, once all scheduled matches for a round are completed, the player will receive an HP deduction of 90% of the average HP lost by players with similar aggregate match scores (+/- 1).
```

i assume this is calculated after every round since you used "each". are the byes counted individually? so if you receive two byes in the same round, you'd lose this 90% twice?

does "aggregate" mean that the match score counted here is the total match score for the stage, or is it just for the round?

Author: Committed (https://github.com/ppy/osu-wiki/pull/7272/commits/2ef651d242bf2ede9a99eb38204bcfe4f19b105c) the changes discussed here with some minor clarifications, if there are any changes that you think are required let me know.

- Due to the nature of the elimination system, byes are unavoidable.

> - If you receive a bye, once the matches for each round are completed you will receive a “tax” of 90% of the average HP lost by the players at your general aggregate match score (+/- 1).

Reviewer: ```suggestion
- If you receive a bye, once the matches for each round are completed you will receive a "tax" of 90% of the average HP lost by the players at your general aggregate match score (+/- 1).
```

# UK Community Cup X

> ![UKCCX banner](img/banner.jpg)

Reviewer 3: this can probably be higher quality and lower filesize if png

| **Jemzsee** ![][flag_GB] | **7** | 1 | ![][flag_GB] thereal99gamer | [#1](https://osu.ppy.sh/community/matches/75768699) |
| Apo11o ![][flag_GB] | 3 | **5** | ![][flag_GB] **[Nathan]** | [#1](https://osu.ppy.sh/community/matches/75751035) |
| **Chemistry** ![][flag_GB] | **7** | 1 | ![][flag_GB] idealnettles | [#1](https://osu.ppy.sh/community/matches/75757300) |
| **Malteser** ![][flag_GB] | **6** | 2 | ![][flag_GB] Matt4132 | [#1](https://osu.ppy.sh/community/matches/75750782) |
| Mahmood ![][flag_GB] | 3 | **5** | ![][flag_GB] **Zyphox** | [#1](https://osu.ppy.sh/community/matches/75776424) |
| **LilyFlower** ![][flag_PH] | **5** | 3 | ![][flag_GB] Best459 | [#1](https://osu.ppy.sh/community/matches/75757514) |
| **Doomsday** ![][flag_GB] | **8** | 0 | ![][flag_GB] squarercube | [#1](https://osu.ppy.sh/community/matches/75744684) |
| **Defex** ![][flag_GB] | **5** | 3 | ![][flag_GB] Taiga Tiger | [#1](https://osu.ppy.sh/community/matches/75747431) |
| **Ophiz** ![][flag_IE] | **6** | 2 | ![][flag_GB] Fwuffy | [#1](https://osu.ppy.sh/community/matches/75757629) |

> | Rinnu ![][flag_GB] | -1 | -1 | ![][flag_GB] _Snowy | *forfeit* |

Reviewer 3: what does it mean for both players to forfeit? or is this a typo?

Reviewer: probably that both players couldn't make it, in which cases they both forfeited their match

- In order to ensure that as many matches take place as possible, reschedules are allowed up to **2 hours before your match is due to be played**. However, your opponent holds the right to **refuse a reschedule if asked less than 12 hours before the scheduled time**.
- If it is impossible to find a time to reschedule to that suits both teams, both teams should talk to the hosts (![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) or ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973)) who will attempt to sort something out.

### Pre-match

- Referees will create the lobby approximately **10 minutes** before the match is due to start.
- Players have a 10-minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will receive losses.
- We take no-shows seriously. If you fail to show up to your match without valid reason, you may forfeit your right to participate in future UKCC tournaments.
- The referee cannot play in the match.

### Qualifiers

- Players will play each map in the Qualifiers mappool once.

> - There will be 3 NoMod, 2 Hidden, 2 Hard Rock and 2 Double Time maps.

Reviewer: ```suggestion
- There will be 3 NoMod, 2 Hidden, 2 Hard Rock, and 2 Double Time maps.
```

> - The top 32 scores will progress into the Swiss bracket.

Reviewer: ```suggestion
- The players with the top 32 scores will progress into the Swiss bracket.
```

> ### Swiss

Reviewer: ```suggestion
### Swiss bracket
```

- Since Swiss bracketing is a natural seeding system, matchups will be drawn between **top 16 and bottom 16 players randomly** to begin.
- Two matches will happen every weekend for each player, one on Saturday and one on Sunday—therefore, **two matchups will be drawn per weekend** for each player.
- The order the matches are played in **does not matter**.
- Matchups will be drawn between players with the **same aggregate match W-L-D score** (eg. players at 3-0-1 will prioritise playing against the players with the most similar W-L-D), and will prioritise **avoiding rematches** until each active player has played each other once.

### Rolling

- When both players are in the lobby and ready, the referee will ask the players to `!roll`. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).

### Banning

- Each player will have **one** ban.

> - Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.

Reviewer: ```suggestion
- Players have **two minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
```

- There are no restrictions on bans and picks. Players are free to ban/pick any map regardless of mod.

### Warmups

- There will be **no** warmups played.

### Picking

- Players have **90 seconds** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.

> - **Failed scores will count, and every map will be played with the No Fail mod enabled**

Reviewer: ```suggestion
- **Failed scores will count, and every map will be played with the No Fail mod enabled.**
```

> - Each player picks **four times**. Once 8 maps have been picked and played, the match ends.

Reviewer: ```suggestion
- Each player picks **four times**. Once eight maps have been picked and played, the match ends.
```

> - There are **no tiebreakers.**

Reviewer: ```suggestion
- There are **no tiebreakers**.
```

### Health system

> - Each player who qualifies into Swiss will **begin with 8,000,000 HP**, which carries through matches.

Reviewer: ```suggestion
- Each player who qualifies into the Swiss bracket will **begin with 8,000,000 HP**, which carries throughout matches.
```

- If a player's HP reaches 0, they are **immediately eliminated from the tournament**, and if their match is still in progress it ends immediately.

### Pocket Picks

- When signing up to the tournament, each player will **submit three maps** to be listed as "Pocket Picks" for use during Swiss.
- Pocket Picks can be **any map, with any mods**. The mods chosen are forced.
- While there is room to push boundaries, the **rule of common sense** applies when submitting Pocket Picks—the maps will be reviewed case by case, and if a map is too far out of the playable range it will be rejected and a new Pocket Pick will have to be submitted.
- Only up to one Pocket Pick can be played per match, and only as a player's **3rd or 4th pick during the match**.
- Once a Pocket Pick has been played, it **cannot be picked again for the rest of the tournament**.
- Pocket Picks cannot be banned.

### Byes

- A player receives a bye when an opponent is eliminated from the tournament before the scheduled match can take place.
- If a player forfeits a match, both players will receive a bye for that match.

> - For each bye a player receives, once all scheduled matches for a round are completed, the player will receive an HP deduction of 90% of the average HP lost by players with similar aggregate match scores (+/- 1) from that round.

Reviewer: ```suggestion
- For each bye a player receives, once all scheduled matches for a round are completed, the player will receive an HP deduction of 90% of the average HP lost by players with similar aggregate match scores (±1) from that round.
```
